src/wp-includes/rest-api/endpoints/class-wp-rest-post-types-controller.php

```diff
@@ -385,6 +390,12 @@ public function get_item_schema() {
 						),
 					),
 				),
+				'icon'           => array(
+					'description' => __( 'The icon for the post type.' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit', 'embed' ),
```

Comment:
Does this need to be in the view context? The icon is not public right now?

Reply:
What do you mean public? I think it would be good to have in `view` as it can be useful in some cases. Do you see any drawback being in `view` context?

Reply:
If you make a property as view it means it can't be viewed pubilically. Do we want to expose this data publically?

Reply:
I see no harm in it, but if you have a strong opinion on this, I can remove it.